'R: Push Release Notes' failing on push

https://github.com/checkstyle/checkstyle/actions/runs/3580170024/jobs/6022052073
```
Run git config user.name 'github-actions[bot]'
[master 4cb1649] doc: release notes for 10.5.1
 1 file changed, 1[7](https://github.com/checkstyle/checkstyle/actions/runs/3580170024/jobs/6022052073#step:5:8) insertions(+)
remote: Permission to checkstyle/checkstyle.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/checkstyle/checkstyle/': The requested URL returned error: 403
Error: Process completed with exit code 12[8](https://github.com/checkstyle/checkstyle/actions/runs/3580170024/jobs/6022052073#step:5:9).
```

We need to define write permission to action.
https://github.com/checkstyle/checkstyle/blob/77040f2cce4767fbb49e60073203a85885c2e99c/.github/workflows/bump-version-and-update-milestone.yml#L13

--- a/.github/workflows/update_sources.yml
+++ b/.github/workflows/update_sources.yml
@@ -13,6 +13,8 @@ jobs:
   releasenotes:
     name: Push releasenotes ${{ github.event.inputs.version }}
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
       - name: Checkout the latest code
         uses: actions/checkout@v3

